src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -224,7 +224,7 @@ public override int Run(AnalyzeOptions analyzeOptions)
             // interesting (it will always be set).
 
             return analyzeOptions.RichReturnCode == true
-                ? (int)((uint)result & ~(uint)RuntimeConditions.RuleNotApplicableToTarget)
+                ? (int)((uint)result & ~(long)RuntimeConditions.RuleNotApplicableToTarget)
                 : result;
```

Comment:
It looks like result is an int and needs to use long. It looks like the long needs to be propagated back up, otherwise the new higher bits are truncated. Then there would not need to be any type casts here.

Reply:
This changeset 1. added a test 2. bringing in the latest SDK to get my fix for "This operation is not supported for a relative URI". 
This issue looks like comes from a checkin before my fix: https://github.com/microsoft/sarif-sdk/pull/2660
I have leave a message to get Michael's input.
@suvamM let me know how do we like to proceed from here,